### PR TITLE
Adding proxy send timeout for the nginx server configuration.

### PR DESCRIPTION
**Pull request by Nitish-SH**

### CODE REVIEW

File: manifests/resource/server.pp [hunk 1]
```diff
@@ -25,6 +25,7 @@
 #   [*proxy*]                      - Proxy server(s) for the root location to connect to.  Accepts a single value, can be used in
 #     conjunction with nginx::resource::upstream
 #   [*proxy_read_timeout*]         - Override the default the proxy read timeout value of 90 seconds
+#   [*proxy_send_timeout*]         - Override the default the proxy read timeout value of 90 seconds
```
Comment (eputnam) [inline manifests/resource/server.pp]: I'm assuming this should read: "Override the default proxy send timeout value of 90 seconds"?

**Comment by Nitish-SH** [comment]
yes this is a type let me fix it and push again.

File: manifests/resource/server.pp [hunk 2]
```diff
@@ -24,7 +24,8 @@
 #   [*autoindex*]                  - Set it on 'on' or 'off 'to activate/deactivate autoindex directory listing. Undef by default.
 #   [*proxy*]                      - Proxy server(s) for the root location to connect to.  Accepts a single value, can be used in
 #     conjunction with nginx::resource::upstream
-#   [*proxy_read_timeout*]         - Override the default the proxy read timeout value of 90 seconds
+#   [*proxy_read_timeout*]         - Override the default proxy read timeout value of 90 seconds
```
Comment (Nitish-SH) [inline manifests/resource/server.pp]: the message is now changed to "Override the default proxy read timeout value of 90 seconds"

Comment (Nitish-SH) [inline manifests/resource/server.pp]: the message is now changed to "Override the default proxy send timeout value of 90 seconds"

**Comment by Nitish-SH** [comment]
should i create new pull request for the typo fixes?

**Comment by eputnam** [comment]
thanks @Nitish-SH !

**Pull request merged**